os-server/src/main/java/app/openschool/category/CategoryServiceImpl.java

```diff
@@ -156,4 +162,11 @@ public void delete(Long categoryId, Locale locale) {
     fileStorageService.deleteFile(oldFileName);
     categoryRepository.deleteById(categoryId);
   }
+
+  private void trowExceptionWhenTitleIsPresent(String title) {
```

Comment:
which error response is returned in this case, better to create a new custom exception and return 400 error with meaningful  error response